resultshandling/printer/summeryhelpers.go

```diff
@@ -29,7 +29,7 @@ func groupByNamespaceOrKind(resources []WorkloadSummary, status func(workloadSum
 }
 
 func isKindToBeGrouped(kind string) bool {
-	if kind == "Group" || kind == "User" {
+	if kind == "Group" || kind == "User" || kind == "CloudProvider" || kind == "ClusterDescription" {
```

Comment:
`CloudProvider` Is a kind we do not support.
Please refer to the [const](https://github.com/armosec/k8s-interface/blob/master/cloudsupport/cloudproviderconfiguration.go#L27) and not `"ClusterDescription"`

Reply:
Otherwise it will be printed as 'namespace'